irohad/main/application.cpp

```diff
@@ -128,13 +129,37 @@ Irohad::Irohad(
       yac_init(std::make_unique<iroha::consensus::yac::YacInit>()),
       consensus_gate_objects(consensus_gate_objects_lifetime),
       log_manager_(std::move(logger_manager)),
-      log_(log_manager_->getLogger()) {
+      log_(log_manager_->getLogger()),
+      on_block_subscription_(std::make_shared<OnBlockSubscription>(
```

Comment:
Please use the same name everywhere `OnBlockSubscription` or `BlockSubscription`

Reply:
https://imgur.com/3l4KntA.png